packages/documentation-site/patternfly-docs/content/developer-resources/rtl-handbook.md

```diff
@@ -0,0 +1,44 @@
+---
+id: Right to left handbook
+title: PatternFly Right to Left (RTL) handbook
```

Comment:
```suggestion
title: PatternFly right-to-left handbook
```

Would recommend avoiding acronyms in headings!

Reply:
I don't think I've ever sees it hyphenated but now I'm doubting myself lol. I made some updates hopefully incorporating all these comments - will revisit tomorrow to make sure I got everything!

Reply:
I'll defer to what you decide is correct since it's not a concept I'm personally familiar with! I saw it once on Material and also on Apple: https://developer.apple.com/design/human-interface-guidelines/right-to-left, but I also see it without hyphens across some websites when I Google 🧐.